src/python/e2e-test-runner/e2e_test_runner/test_engagement_edge.py

```diff
@@ -0,0 +1,32 @@
+"""
+TODO (wimax July 2020): I don't see anything in here that indicates that
+screams "e2e test", this certainly seems like more of an integration test.
+
+There's nothing here that does anything cross-service.
+Perhaps it's just "does it work in AWS?"
```

Comment:
Yes, I believe that's the case. I'd like to merge the e2e tests and integration tests into just "integration tests" at some point, and make them all run in AWS.